docs/advisories/meltdown-kernel-update.md

```diff
@@ -56,39 +52,46 @@ For the kops-maintained AMIs, the following AMIs contain an updated kernel:
 - kope.io/k8s-1.8-debian-stretch-amd64-hvm-ebs-2018-01-05
 - kope.io/k8s-1.8-debian-stretch-amd64-hvm-ebs-2018-01-05
 
-These are the images that are maintained by the kops project; please refer to
+These are the images that are maintained by the kubernetes/kops project; please refer to
 other vendors for the appropriate AMI version.
 
 ### Update Process
 
 For all examples please replace `$CLUSTER` with the appropriate kops cluster
 name.
 
-#### Determine which instance groups exist
+#### List instance groups
 
 `kops get ig --name $CLUSTER`
 
-#### Edit the kops instance groups 
+#### Replace the image for each instance group
```

Comment:
They are not replacing.  Maybe the wrong verb?  They are modifying the cluster???

Reply:
nit pick btw

Reply:
Changing to `Update the image for each instance group`